tests/src/main/java/io/openliberty/tools/eclipse/test/it/AbstractLibertyPluginSWTBotTest.java

```diff
@@ -50,7 +50,7 @@
 import org.junit.jupiter.api.TestInfo;
 
 import io.openliberty.tools.eclipse.test.it.utils.LibertyPluginTestUtils;
-import static io.openliberty.tools.eclipse.DevModeOperations.MVN_RUN_APP_LOG_FILE;
+import jdk.nashorn.tools.Shell;
```

Comment:
Looks like this snuck in by mistake.

Reply:
yup.... just noticed this failing in GHA